### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by mklewandowski**
Add release steps to readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,15 @@ npm publish --access=public
...
 
+#### Production Release
+To deploy a production release:
+
+1. Merge all changes to `master`
+2. Use https://github.com/concord-consortium/portal-report/releases to create a new release tag
+3. Login to https://learn.staging.concord.org/ as an admin and under Admin > Site Admin Links > External Reports update the URL field of any External Reports that use the latest version of portal-report. Change the URL to the newly created release (e.g., https://portal-report.concord.org/version/vx.y.z/).
+4. Run any QA and testing on staging release.
+5. Once testing is complete, login to https://learn.concord.org/ as an admin and under Admin > Site Admin Links > External Reports update the URL field of any External Reports that use the latest version of portal-report. Change the URL to the newly created release (e.g., https://portal-report.concord.org/version/vx.y.z/).
+
```
Comment (tejal-shah) [inline README.md:93]: @mklewandowski , We probably should add a Step 6 to test in production once the URL in learn.concord.org is updated.

Comment (kswenson) [reply README.md:93]: Why bother? If you've made it this far I'm _sure_ it's fine. 😉

Comment (mklewandowski) [reply README.md:94]: Sounds good!  Step added.

Comment (mklewandowski) [reply README.md:94]: Anyone is welcome to approve this so we can merge.

Comment (tejal-shah) [reply README.md:94]: I'd be happy to approve but it looks like only @eireland can. I can only add comments to this PR as far as I can tell.

Comment (mklewandowski) [reply README.md:94]: @tejal-shah I added you as a reviewer.

File: README.md [hunk 2]
```diff
@@ -82,6 +82,16 @@ npm publish --access=public
 
 NPM will automatically build the library first, as `package.json` defines `script/prepare`.
 
+#### Production Release
+To deploy a production release:
+
+1. Merge all changes to `master`
+2. Use https://github.com/concord-consortium/portal-report/releases to create a new release tag
```
Comment (scytacki) [inline README.md:89]: At least in the past this approach hasn't been safe. When you make a new release tag directly in GitHub it makes a lightweight tag. And our s3-deploy code can get confused (at least when it was running on travis) by lightweight tags.

So to be safe you should make a tag in your local git client (commandline or GUI) that has a description associated with it. I usually put at least the version number in this description. Then push this tag to production. Now after making this "heavy weight" tag you can go to Github and make a release and choose this tag.

**Review by scytacki — CHANGES_REQUESTED** [review]
github makes lightweight tags which can be problematic.

File: README.md (hunk 2 above)
Comment (mklewandowski) [reply README.md:89]: This is straight from the starter project so if this is true, then most of our documents release procedures are wrong:
https://github.com/concord-consortium/starter-projects#deployment

Comment (mklewandowski) [reply README.md:89]: So is the "annotated" tag what you are referring to?
https://git-scm.com/book/en/v2/Git-Basics-Tagging

Comment (mklewandowski) [reply README.md:89]: Is this what you're thinking?
```
3. Make a new version tag using your local git client with the version number in the description (e.g., `git tag -a v1.1.1 -m "version 1.1.1"`) and push the tag to the portal-report repo (e.g., `git push origin v1.1.1`).
4. Use https://github.com/concord-consortium/portal-report/releases to create a new release using the newly created tag.
```

Comment (scytacki) [reply README.md:89]: Yes an annotated tag is correct. 

The starter projects description uses a production branch, so it doesn't matter much if the tag gets deployed or not. But for something like the portal-report or question interactives it does mater that the tag is deployed correctly. I thought I had corrected this in the starter projects before. I do try to point it out whenever people talk about using Github to create tags.

Comment (mklewandowski) [reply README.md:89]: Makes sense.  I'll push the above changes.

Comment (mklewandowski) [reply README.md]: Steps updated.  See the latest commit.

Comment (mklewandowski) [reply README.md]: Codecov doesn't like the lack of readme tests! 🤣

**Review by scytacki — APPROVED** [review]
looks good to me.

File: README.md (hunk 2 above)
Comment (scytacki) [reply README.md]: :) It is actually that there is a random difference when cypress runs the tests. Sometimes it causes a scroll event and sometimes it doesn't: https://app.codecov.io/gh/concord-consortium/portal-report/compare/335/changes

So when the scroll event doesn't happen not as much code is covered.

**Pull request merged**